apps/condo/domains/organization/schema/OrganizationEmployee.test.js

```diff
@@ -231,4 +237,91 @@ describe('OrganizationEmployee', () => {
             await updateTestOrganizationEmployee(userClient, obj.id, { name: 'name3' })
         })
     })
+
```

Comment:
You need to add a test for after change hook

Reply:
it is tested in these tests, for example in test 'employee who accepted the invite: updates phone and email if employee user updates' I tested that when updating the user we updates relates employee (in after change hook)